xml/System.Security.Cryptography/ECDiffieHellmanOpenSsl.xml

```diff
@@ -54,9 +72,26 @@
         <Parameter Name="keySize" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="keySize">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="keySize">The size of the key to generate, when a key is needed.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Security.Cryptography.ECDiffieHellmanOpenSsl" /> class defaulting to the NIST prime curve of the specified size.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Only three key sizes are supported by this constructor: 256 (NIST P-256/secp256r1), 384 (NIST P-384/secp384r1), and 521 (NIST P-521/secp521r1).
+To generate keys For any other curve, use the <xref:System.Security.Cryptography.ECDiffieHellmanOpenSsl.%2Ector(System.Security.Cryptography.ECCurve)> constructor
```

Comment:
Malformed xref. I think it's the `%2E` substituting the dot:

```suggestion
To generate keys For any other curve, use the <xref:System.Security.Cryptography.ECDiffieHellmanOpenSsl..ctor(System.Security.Cryptography.ECCurve)> constructor
```

Reply:
@mairaw @BillWagner  Is this the right way to mention constructors in xrefs? I noticed the build warnings are also saying this is not correct:

```xml
<xref:System.Security.Cryptography.ECDiffieHellmanOpenSsl..ctor(System.Security.Cryptography.ECCurve)>
```

Reply:
I'll suggest this one, if it doesn't work we can correct it:
```suggestion
To generate keys For any other curve, use the <xref:System.Security.Cryptography.ECDiffieHellmanOpenSsl..Ector(System.Security.Cryptography.ECCurve)> constructor
```

Reply:
Here's some info on the link to apis:
https://github.com/dotnet/docs/blob/master/styleguide/template.md#links-to-apis

I'll take a look at those. You have to encode the # character with %23